kong/db/schema/others/declarative_config.lua

```diff
@@ -907,7 +980,9 @@ local function load_entity_subschemas(entity_name, entity)
 end
 
 
-function DeclarativeConfig.load(plugin_set, vault_set, include_foreign)
+-- @tparam sync_v2_enabled It generates full schema and foreign references to
+-- validate schema and references for sync.v2
+function DeclarativeConfig.load(plugin_set, vault_set, include_foreign, sync_v2_enabled)
```

Comment:
Could we move `sync_v2_enabled` into the function body?

Reply:
seem s ok

Reply:
no, no, from the function body inside, we could not know where the function is called , might be called from kong/init, or otherplace. if other place, i think the kong.sync logic might not exists. so if you check kong.sync is weired there.